PlayKitSmartSwitch/Classes/SmartSwitchPluginConfig.swift

```diff
@@ -21,18 +21,29 @@ import Foundation
     // Your YOUBORA account code.
     @objc public var accountCode: String = ""
     
-    // CDN group configured to select a subset of configured CDNs.
-    // vod, live1, live2
-    @objc public var originCode: String = ""
+    // The application code configured in the UI
+    @objc public var application: String?
     
+    /*
+    // Set any of these additional parameters:
+     ip,
+     originCode,
+     userAgent,
+     live,
+     protocol,
+     extended,
+     nva (as a String),
+     nvb (as a String),
+     token
+     */
     @objc public var optionalParams: [String: String]?
     
     @objc public var timeout: TimeInterval = 10
     
     @objc public var reportSelectedCDNCode: Bool = false
     
-    //SmartSwitch server url
-    @objc public var smartSwitchUrl: String = "http://cdnbalancer.youbora.com/orderedcdn"
+    // SmartSwitch domain url default: https://api.gbnpaw.com
```

Comment:
I advise to mention the full url in the documentation.
And mention that the values of the accountCode and application are taken from the config.

```suggestion
    // SmartSwitch domain url default: https://api.gbnpaw.com/{accountCode}/{application}/decision
```